src/components/EmojiPicker/CategoryShortcutButton.js

```diff
@@ -1,4 +1,4 @@
-import React, {PureComponent} from 'react';
```

Comment:
Given that this was using PureComponent before. We should compose it with React.memo so that we still have the same optimization.